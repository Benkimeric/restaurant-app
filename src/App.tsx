import React from "react";
import MainMenu from "./components/MainMenu";

const App = () => {
  return (
    <div className="App">
      <header className="App-header">
        <MainMenu />
      </header>
    </div>
  );
};

export default App;
