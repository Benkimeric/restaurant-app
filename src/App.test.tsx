import React from "react";
import Enzyme, { mount } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import App from "./App";

Enzyme.configure({ adapter: new Adapter() });

test("renders APP without crashing", () => {
  const wrapper = mount(<App />);

  expect(wrapper).toMatchSnapshot();
});
