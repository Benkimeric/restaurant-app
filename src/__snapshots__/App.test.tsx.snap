// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders APP without crashing 1`] = `ReactWrapper {}`;
